runtime/jcl/common/java_lang_Class.cpp

```diff
@@ -1847,7 +1847,6 @@ Java_java_lang_Class_getNestMembersImpl(JNIEnv *env, jobject recv)
 
 	if (NULL == nestHost) {
 		if (J9_VISIBILITY_ALLOWED != vmFuncs->loadAndVerifyNestHost(currentThread, clazz, 0)) {
-			nestMember = clazz;
```

Comment:
Just a minor comment, since `nestMember` is not needed here, `nestMember` can be declared closer to its usage after `nestMemberName` declaration.

Reply:
Unfortunately it can't as there are gotos in the code that the compiler errors on due to jumping over initialization

Reply:
Is that `Windows` specific? `Linux x86 AMD gcc 7.3.0` in my docker compiles without warning/error though.

Reply:
You're right.  I had shortened the lifespans of several variables and it was the others that were causing the issue.  Updated this one.